crates/pop-cli/src/commands/install/mod.rs

```diff
@@ -2,32 +2,234 @@
 
 use anyhow::Context;
 use clap::Args;
+use cliclack::{confirm, log, outro};
+use duct::cmd;
+use os_info::Type;
 use tokio::{fs, process::Command};
 
 #[derive(Args)]
 #[command(args_conflicts_with_subcommands = true)]
-/// Setup user environment for substrate development
-/// Runs script `scripts/get_substrate.sh`
-pub(crate) struct InstallArgs;
+/// Setup user environment for development
+pub(crate) struct InstallArgs {
+	/// Before install all the dependencies needed, do not ask the user for confirmation.
+	#[clap(short('y'), long)]
+	skip_confirm: bool,
+}
 
 impl InstallArgs {
 	pub(crate) async fn execute(self) -> anyhow::Result<()> {
-		if cfg!(target_os = "windows") {
-			return Ok(cliclack::log::error("Windows is supported only with WSL2")?);
+		if cfg!(target_os = "macos") {
```

Comment:
Missing https://github.com/r0gue-io/pop-cli/blob/ccfc12322d8d7da61c81d87568854517ddcedcf7/crates/pop-cli/src/commands/new/parachain.rs#L71-L72 and https://github.com/r0gue-io/pop-cli/blob/ccfc12322d8d7da61c81d87568854517ddcedcf7/crates/pop-cli/src/commands/new/parachain.rs#L99

Reply:
Good point